workshops/dodge/README.md

```diff
@@ -298,16 +327,16 @@ This is our lose condition. Now we can punish the user for losing in this `gameO
 
 ### Game Over
 
-Even though it may feel like we've been writing all of our code in these two special p5js functions to make stuff in the canvas, we musn't forget that we are still just writing JavaScript. So let's define our own function, `gameOver()`, the way we would any other JS function. We'll place it underneath the `draw()` function.
+Even though it may feel like we've been writing all of our code in these two special p5.js functions to make stuff in the canvas, we musn't forget that we are still just writing JavaScript. So let's define our own function, `gameOver()`, the way we would any other JS function. We'll place it underneath the `draw()` function.
 
-```
+```js
 function gameOver() {
 }
 ```
 
-And let's display a Game Over screen.
+And let's display a game over screen.
```

Comment:
Well, "Game Over" is not a proper noun. So shouldn't be capitalized. Not
sure why we'd style it as game-over either.

On Wed, Apr 6, 2016 at 10:39 AM, JESSICA <notifications@github.com> wrote:

> In workshops/dodge/README.md
> <https://github.com/jkwok91/hackclub/pull/2#discussion_r58748832>:
>
> >  function gameOver() {
> >  }
> >  ```
> >
> > -And let's display a Game Over screen.
> > +And let's display a game over screen.
>
> Veto. Either "Game Over" or game-over
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/jkwok91/hackclub/pull/2/files/f24ad8246cdf7e3e53e56b84653a167ecb9fe63a#r58748832>
>



-- 
Zach Latta
https://zachlatta.com <http://zachlatta.com>